.lintstagedrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "!(.github/workflows/*.yml|*.js|package.json)": [
+    "prettier --ignore-unknown --write"
+  ],
+  ".github/workflows/*.yml": ["npx .", "prettier --write"],
```

Comment:
We should include local actions. See, for example,e in the [ui-monorepo](https://github.com/hemilabs/ui-monorepo/tree/main/.github/actions)